### DISCUSSION TITLE
给洛雪音乐助手的一个发展思路

### DISCUSSION THREAD

**Discussion by Phrixus2023** [lyswhut/lx-music-desktop#1736] [Ideas]
目前由于中心化思路导致这个项目会遭到大厂的围剿，能否尝试去中心化策略来实现该项目路线，去中心化代表项目IPFS，感觉可以利用IPFS路线来实现音乐程序，资源全部分散到全球各地的电脑里，这样软件本身并不提供资源，只是提供资源的连接。每一个人都为此项目贡献资源。https://github.com/ipfs/ipfs-desktop

**Comment by chaohua27** [comment]
P2P目前打击的更狠。

**Reply by liya558** [reply]
上一个这样做的人叫王欣，刚出来

**Comment by suzikuo** [comment]
为什么打击快播？因为如果人们有了依赖性，那么版权意识就没有了。
版权意识没有了，盗版泛滥，再也没有创造者了。所以才大力打击。
只能在一小部分人之间使用
